client/web/compose/src/components/Namespaces/Reminders/Edit.vue

```diff
@@ -146,7 +146,9 @@ export default {
   components: {
     VueSelect,
     CInputDateTime,
+    CButtonSubmit,
```

Comment:
If you register it globally (like you did and should for all apps). Then you don't need to import it anywhere